### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by CHENYUANYUAN1999**

### CODE REVIEW

**Comment by CHENYUANYUAN1999** [comment]
@LiJianhong-qwq

**Comment by CHENYUANYUAN1999** [comment]
@LiJianhong-qwq

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+def get_user_info():
```
Comment (LiJianhong-qwq) [inline prac_05/emails.py:1]: no main() fucntion

File: prac_05/hex_colours.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "blanchedalmond": "#FFEBCD",
+    "blue": "#0000FF"
+}
+
+while True:
+    color_name = input("Enter a color name: ").lower()
+    if color_name in COLOR_OF_CODE:
+        print(COLOR_OF_CODE[color_name])
+    else:
+        print("Invalid color name")
```
Comment (LiJianhong-qwq) [inline prac_05/hex_colours.py:19]: This loop is endless.

File: prac_05/wimbledon.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import csv
```
Comment (LiJianhong-qwq) [inline prac_05/wimbledon.py:1]: It is not required to use this module.

**Comment by LiJianhong-qwq** [comment]
Don't use something that the course doesn't teach.

**Pull request merged**